src/content/structured/components/badge/guidance.mdx

```diff
@@ -0,0 +1,243 @@
+---
+path: "/components/badge"
```

Comment:
Code examples have a self closing tag for web components e.g. <ic-badge/> when they should have a closing tag e.g. <ic-badge> </ic-badge> for web components (not for React)